tests/components/schlage/test_select.py

```diff
@@ -0,0 +1,13 @@
+"""Test Schlage select."""
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+
+async def test_select(
```

Comment:
The state test can use snapshots (https://developers.home-assistant.io/docs/development_testing/#snapshot-testing). But you should also add a test for changing the state, ensuring it calls the Schlage API properly.

Reply:
I will look into this. For some reason when I run `pytest tests/components/schlage/test_init.py --snapshot-update` the snapshot doesn't update even though I've added a new value for `auto_lock_time`

Reply:
I've updated the test to set the option based on `demo/test_select.py`